runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -144,7 +144,7 @@ def runnerV2PipelineOptions = [
   "--region=${dataflowRegion}",
   "--tempRoot=${dataflowValidatesTempRoot}",
   "--workerHarnessContainerImage=${dockerImageContainer}:${dockerTag}",
-  "--experiments=beam_fn_api,use_unified_worker,use_runner_v2",
+  "--experiments=beam_fn_api,use_unified_worker,use_runner_v2,shuffle_mode=appliance",
```

Comment:
do we need a TODO or something here to re-enable this after we fix the internal bug?

Reply:
we can track this in internal bug.

Reply:
Would you like to elaborate more on why we need this change?

Reply:
it seems that there's some sharding logic misconfiguration with shuffle_mode=service when using runner v2, the issue is tracked internally with b/179694817

Reply:
Thanks for the explanation! I'm curious that do we only have java tests affected or python as well?